pydarn/plotting/acf.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Marian Schmidt
```

Comment:
Marina**

Reply:
Some in the CEDAR workshop kept calling me this and so do you. Maybe I am just changing my name slowly :p